extract.py

```diff
@@ -2,94 +2,102 @@
 import fontforge as ff
 from glob import glob
 
+
```

Comment:
please do not re-format the whole file if you just want to change one line!

Reply:
Then, please share your formatter configuration.